drv/stm32xx-i2c/src/lib.rs

```diff
@@ -923,7 +923,7 @@ impl I2cController<'_> {
 
         #[rustfmt::skip]
         i2c.oar1.modify(|_, w| { w
-            .oa1en().clear_bit()                    // own-address disable 
```

Comment:
**note:** Zed editor did these format fixes automatically, so they're kind of "slipping in" on the side.